test/message/test_runner_output_spec_reporter_with_coverage.out

```diff
@@ -0,0 +1,287 @@
+
+ custom inspect symbol fail (*ms)
+  customized
+
+ custom inspect symbol that throws fail (*ms)
+  { foo: 1, [Symbol(nodejs.util.inspect.custom)]: [Function: [nodejs.util.inspect.custom]] }
+
+ subtest sync throw fails
+   sync throw fails at first (*ms)
+    Error: thrown from subtest sync throw fails at first
+    *
+    *
+    *
+    *
+    *
+    *
+    *
+    *
+    *
+    *
+
+   sync throw fails at second (*ms)
+    Error: thrown from subtest sync throw fails at second
+    *
+    *
+    *
+    *
+    *
+    *
+    *
+    *
+    *
+    *
+
+ subtest sync throw fails (*ms)
+
+ timed out async test (*ms)
+  'test timed out after *ms'
+
+ timed out callback test (*ms)
+  'test timed out after *ms'
+
+ large timeout async test is ok (*ms)
+ large timeout callback test is ok (*ms)
+ successful thenable (*ms)
+ rejected thenable (*ms)
+  'custom error'
+
+ unfinished test with uncaughtException (*ms)
+  Error: foo
+  *
+  *
+  *
+
+ unfinished test with unhandledRejection (*ms)
+  Error: bar
+  *
+  *
+  *
+
+ invalid subtest fail (*ms)
+  'test could not be started because its parent finished'
+
+ Warning: Test "unhandled rejection - passes but warns" generated asynchronous activity after the test ended. This activity created the error "Error: rejected from unhandled rejection fail" and would have caused the test to fail, but instead triggered an unhandledRejection event.
+ Warning: Test "async unhandled rejection - passes but warns" generated asynchronous activity after the test ended. This activity created the error "Error: rejected from async unhandled rejection fail" and would have caused the test to fail, but instead triggered an unhandledRejection event.
+ Warning: Test "immediate throw - passes but warns" generated asynchronous activity after the test ended. This activity created the error "Error: thrown from immediate throw fail" and would have caused the test to fail, but instead triggered an uncaughtException event.
+ Warning: Test "immediate reject - passes but warns" generated asynchronous activity after the test ended. This activity created the error "Error: rejected from immediate reject fail" and would have caused the test to fail, but instead triggered an unhandledRejection event.
+ Warning: Test "callback called twice in different ticks" generated asynchronous activity after the test ended. This activity created the error "Error [ERR_TEST_FAILURE]: callback invoked multiple times" and would have caused the test to fail, but instead triggered an uncaughtException event.
+ Warning: Test "callback async throw after done" generated asynchronous activity after the test ended. This activity created the error "Error: thrown from callback async throw after done" and would have caused the test to fail, but instead triggered an uncaughtException event.
+ tests 65
+ pass 27
+ fail 21
+ cancelled 2
+ skipped 10
+ todo 5
+ duration_ms *
+ start of coverage report
+ *
+ *
+ *
+ *
+ *
+ end of coverage report
```

Comment:
It seems like we're not validating the important parts here. You can see how I tested this [here](https://github.com/nodejs/node/commit/8c95cb012b4c9c5b304cd8fbc191b4677a20ec98#diff-be4e61fa44baef0cc520813c5250d162ab5a7da4c778c977829cb559f7323c1cR21).